src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -16,395 +22,258 @@ servers:
 
 paths:
 
-  # TODO: Sample endpoint. Remove when building a concrete application
-  # This definition uses parameters: http://example.com/myservice/v1/hello?alternateHello=Boston
-  /hello:
+  /IIP:
```

Comment:
I think this should be directly under wherever the service is deployed. I guess `/IIP:` should be just `/:`